RELEASE_NOTES.md

```diff
@@ -30,3 +30,5 @@ _Note: Remember to restart the AppDaemon addon/server after updating to a new ve
 
 - [XYZ](https://BASE_URL/controllerx/controllers/XYZ) - add device with Z2M support. [ #123 ]
```

Comment:
Hi @Langthjem , thank you for the PR :)

Could you please follow this format?

```
- [E2002](https://BASE_URL/controllerx/controllers/E2002) - add Media Player support for ZHA.
```

Also, could you please uncomment the `<!--` and `-->` around the "New devices" section? So the title is shown together with this item?

Thank you,
Xavi M.